src/scrapy_settings_log/__init__.py

```diff
@@ -51,6 +51,16 @@ def spider_closed(self, spider):
         regex = settings.get("SETTINGS_LOGGING_REGEX")
         if regex is not None:
             settings = {k: v for k, v in settings.items() if re.search(regex, k)}
+        if settings.get("MASKED_SENSITIVE_SETTINGS_ENABLED", True):
+            default_regexes = [
```

Comment:
**todo:** This should be defined in a variable outside this class, so it can be imported and extended by projects if needed.
 
**suggestion (non-blocking):** We could have two settings, `MASKED_SENSITIVE_SETTINGS_REGEX_LIST` and `DEFAULT_MASKED_SENSITIVE_SETTINGS_REGEX_LIST`, similar to what is done with scrapy's `*_BASE` settings (https://github.com/scrapy/scrapy/blob/master/scrapy/settings/default_settings.py#L71). The final list will be constructed with the union of both settings. That way, if developers want to add to the list and keep the current defaults, they'd only need to add them to `MASKED_SENSITIVE_SETTINGS_REGEX_LIST` and keep `DEFAULT_MASKED_SENSITIVE_SETTINGS_REGEX_LIST` intact. But this is only a usability concern, it should be alright to discuss it later.

Reply:
As a note here, I'd add `default_regexes` at the top of this existing module (probably renamed to uppercased letters, DEFAULT_REGEXES). I wouldn't create a new file. I'd still include that change in this PR. The other comment we can leave for later.